core/runtime/src/test/java/org/eclipse/sw360/antenna/configuration/ConfigurationTest.java

```diff
@@ -62,7 +62,7 @@ public void test() {
 
         ArtifactIdentifier identifier = new ArtifactFilename("overrideName.jar");
         assertThat(configuration.getFinalLicenses().keySet().stream().anyMatch(k -> k.matches(identifier))).isTrue();
-        List<LicenseInformation> list = configuration.getFinalLicenses().entrySet().stream()
+        List<License> list = configuration.getFinalLicenses().entrySet().stream()
```

Comment:
To the reader who's not so familiar with the code base (like me) it's surprising that this actually just works. It would be nice to read in the commit message *why* this is a fix, and why it "worked" (i.e. compiled) also before, but what's better now.